client/power-zone.ejs

```diff
@@ -64,22 +70,60 @@
                     </div>
                   </div>
                   <div class="form-row">
-                    <div class="col-lg-6">
+                    <div class="col-xl-6">
                       <div class="form-group">
                         <label for="inputFileName">Power Plan File Name</label>
+                        <div class="input-group">
+                          <input
+                            type="text"
+                            class="form-control"
+                            id="inputFileName"
+                            placeholder="Filename"
+                            required
+                          />
+                          <div class="input-group-append">
+                            <span
+                              class="input-group-text"
+                              id="inputFileExtension"
+                              >.json</span
+                            >
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="form-row">
+                    <div class="col-xl-6">
+                      <div class="form-group">
+                        <label for="inputMass">Mass</label>
                         <input
-                          type="text"
+                          type="number"
                           class="form-control"
-                          id="inputFileName"
-                          placeholder="Filename"
+                          id="inputMass"
+                          placeholder="Mass"
+                          min="0"
                           required
                         />
-                        <span>.pkl</span>
                       </div>
                     </div>
                   </div>
                   <div class="form-row">
-                    <div class="col-lg-6">
+                    <div class="col-xl-6">
+                      <div class="form-group">
+                        <label for="inputStartAdjust">Start Adjust</label>
+                        <input
+                          type="number"
+                          class="form-control"
+                          id="inputStartAdjust"
+                          placeholder="Start Adjust"
+                          min="0"

```

Comment:
startadjust can be a negative integer as well

Reply:
Cheers for this. assumed they were positive numbers only. Can you also let me know what the min/max for the other parameters?